package.json

```diff
@@ -9,12 +9,24 @@
     "postinstall": "yarn build",
     "clean": "lerna clean --yes && rm -rf node_modules",
     "build": "lerna run build",
-    "test": "lerna run test"
+    "test": "lerna run test",
+    "lint": "lerna run lint",
+    "precommit": "lint-staged"
   },
   "dependencies": {
     "lerna": "2.9.0"
   },
-  "workspaces": [
-    "packages/*"
-  ]
+  "workspaces": ["packages/*"],
+  "devDependencies": {
```

Comment:
It strikes me that you're missing `devDependencies` on `prettier` and `tslint`, both called from the `lint-staged` config? Otherwise, you're either depending on globally-installed modules, or the implicit provision via yarn workspace-hoisted dependencies.